boards/arc/axs/doc/index.rst

```diff
@@ -0,0 +1,192 @@
+you need to setup the board correctly.
+
+* Connect the digilent USB cable from your host to the board.
+
+* Connect the 12V DC power supply to your board
+
+Building Sample Applications
+==============================
+
+You can try many of the :ref:`sample applications and demos
+<samples-and-demos>`.  We'll use :ref:`hello_world`, found in
+:zephyr_file:`samples/hello_world` as an example.
+
+Configuring
+-----------
+
+You may need to write a prj_arc.conf file if the sample doesn't have one.
+Next, you can use the menuconfig rule to configure the target. By specifying
+``axs`` as the board configuration, you can select the AXS Software
+Development platform support for Zephyr.
+
+.. zephyr-app-commands::
+   :board: axs
+   :zephyr-app: samples/hello_world
+   :goals: menuconfig
+
+
+Building
+--------
+
+You can build an application in the usual way.  Refer to
+:ref:`build_an_application` for more details. Here is an example for
+:ref:`hello_world`.
+
+.. zephyr-app-commands::
+   :board: axs
+   :zephyr-app: samples/hello_world
+   :maybe-skip-config:
+   :goals: build
+
+Connecting Serial Output
+========================
+
+In the default configuration, Zephyr's AXS Software Development platform
+images support serial output via the USB-UART on the board.
+To enable serial output:
+
+* Open a serial port emulator (i.e. on Linux minicom, putty, screen, etc)
+
+* Specify the tty driver name, for example, on Linux this may be
+  :file:`/dev/ttyUSB0`
+
+* Set the communication settings to:
+
+
+========= =====
+Parameter Value
+========= =====
+Baud:     115200
+Data:     8 bits
+Parity:    None
+Stopbits:  1
+========= =====
+
+Debugging
+==========
+
+Using the latest version of Zephyr SDK(>=0.9), you can debug and flash AXS
+Software Development platform directly.
+
+One option is to build and debug the application using the usual
+Zephyr build system commands.
+
+.. zephyr-app-commands::
+   :board: axs
+   :app: <my app>
+   :goals: debug
+
+At this point you can do your normal debug session. Set breakpoints and then
+:kbd:`c` to continue into the program.
+
+The other option is to launch a debug server, as follows.
+
+.. zephyr-app-commands::
+   :board: axs
+   :app: <my app>
+   :goals: debugserver
+
+Then connect to the debug server at the AXS Software Development platform from
+a second console, from the build directory containing the output :file:`zephyr.elf`.
+
+.. code-block:: console
+
+   $ cd <my app>
+   $ $ZEPHYR_SDK_INSTALL_DIR/arc-zephyr-elf/bin/arc-zephyr-elf-gdb zephyr.elf
+   (gdb) target remote localhost:3333
+   (gdb) load
+   (gdb) b main
+   (gdb) c
+
+Flashing
+========
+
+If you just want to download the application to the AXS Software Development
+platform's DDR and run, you can do so in the usual way.
+
+.. zephyr-app-commands::
+   :board: axs
+   :app: <my app>
+   :goals: flash
+
+This command still uses openocd and gdb to load the application elf file to
```

Comment:
Not any longer. Now `flash` loads `zephyr.elf` with help of OpenOCD and then starts execution from the entry-point. That said GDB is no longer used here at all.